.travis.yml

```diff
@@ -234,38 +234,6 @@ jobs:
         - make image-build-helm
         - make image-push-helm
 
-    # Build and deploy arm64 scorecard-proxy docker image
-    - <<: *deploy
-      name: Docker image for scorecard-proxy (arm64)
-      arch: arm64
-      script:
-        - make image-build-scorecard-proxy
-        - make image-push-scorecard-proxy
-
-    # Build and deploy amd64 scorecard-proxy docker image
-    - <<: *deploy
-      name: Docker image for scorecard-proxy (amd64)
-      arch: amd64
-      script:
-        - make image-build-scorecard-proxy
-        - make image-push-scorecard-proxy
-
-    # Build and deploy ppc64le scorecard-proxy docker image
-    - <<: *deploy
-      name: Docker image for scorecard-proxy (ppc64le)
-      arch: ppc64le
-      script:
-        - make image-build-scorecard-proxy
-        - make image-push-scorecard-proxy
-
-    # Build and deploy s390x scorecard-proxy docker image
-    - <<: *deploy
-      name: Docker image for scorecard-proxy (s390x)
-      arch: s390x
-      script:
-        - make image-build-scorecard-proxy
-        - make image-push-scorecard-proxy
-
```

Comment:
:heart: That's one less set of images to fail during a build :)

Reply:
but do you not need these images?

Reply:
the scorecard-proxy is not used in the new scorecard so its fine to remove.